lib/master_manipulator/version.rb

```diff
@@ -1,5 +1,5 @@
 module MasterManipulator
   module Version
-    STRING = '2.0.0'
+    STRING = '2.1.0'
```

Comment:
I think, but am not sure.  That the release process may bump this version for you...
https://confluence.puppetlabs.com/display/QA/Releasing+Master+Manipulator